src/components/pagination/pagination.scss

```diff
@@ -0,0 +1,265 @@
+  align-items: center;
+  display: flex;
+  height: $action-size;
+  justify-content: center;
+  text-decoration: none;
+  user-select: none;
+}
+
+/**
+ * The current state should inherit the parent element's text color to appear
+ * just a bit muted.
+ */
+
+.c-pagination__action[aria-current] {
+  color: inherit;
+}
+
+/**
+ * When an action element has `is-previous` or `is-next` state classes applied,
+ * we absolute-position pseudo elements that provide extra "previous" and "next"
+ * affordances without requiring any additional markup.
+ */
+
+.c-pagination__action.is-previous::before,
+.c-pagination__action.is-next::after {
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: sizes.$control-icon-size;
+  border: sizes.$edge-small solid colors.$gray-light;
+  border-radius: sizes.$radius-full;
+  bottom: 0;
+  content: '';
+  height: $action-size;
+  margin: auto;
+  position: absolute;
+  top: 0;
+  transition: filter motion.$speed-slow motion.$ease-out,
+    transform motion.$speed-quick motion.$ease-out;
+  width: $action-size;
+
+  .t-dark & {
+    background-color: colors.$primary-brand-dark;
+    border-color: transparent;
+  }
+}
+
+/**
+ * These affordances have similar hover and active states to buttons. We apply
+ * these styles relative to the action state (rather than the pseudo element)
+ * so hovering over a page number will _also_ trigger the animation on the
+ * affordance, which helps enforce the action being considered.
+ */
+
+.c-pagination__action.is-previous:hover::before,
+.c-pagination__action.is-next:hover::after {
+  filter: brightness(110%);
+  transform: scale(1.05);
+}
+
+.c-pagination__action.is-previous:active::before,
+.c-pagination__action.is-next:active::after {
+  filter: brightness(80%);
+  transform: scale(0.95);
+}
+
+/**
+ * Direction-specific icons and positioning
+ */
+
+.c-pagination__action.is-previous::before {
+  background-image: svg-load(
+    'icons/arrow-left.svg',
+    color=colors.$primary-brand
+  );
+  left: 0;
+
+  .t-dark & {
+    background-image: svg-load(
+      'icons/arrow-left.svg',
+      color=colors.$text-light-emphasis
+    );
+  }
+}
+
+.c-pagination__action.is-next::after {
+  background-image: svg-load(
+    'icons/arrow-right.svg',
+    color=colors.$primary-brand
+  );
+  right: 0;
+
+  .t-dark & {
+    background-image: svg-load(
+      'icons/arrow-right.svg',
+      color=colors.$text-light-emphasis
+    );
+  }
+}
+
+/**
+ * The inner number element gives us a means of positioning indicator pseudo
+ * elements relative to the action content without disrupting the "previous"
+ * and "next" affordances.
+ *
+ * 1. Constrain the focus ring (not visible initially) to a circle slightly
+ *    smaller than the "previous" and "next" affordances.
+ * 2. Fixes an issue where Chrome renders the digits a pixel or two higher
+ *    than in other browsers.
+ * 3. The aforementioned means of positioning the indicator pseudo element.
```

Comment:
Nit: It might be nice to include more context here. I know what you're alluding to since I've been reading through this file, but if I was a future dev troubleshooting an issue I'd need to dig through this whole file to figure it out

Reply:
So, when I say "the aforementioned means," I'm talking about the preceding paragraph, which says:

> The inner number element gives us a means of positioning indicator pseudo elements relative to the action content without disrupting the "previous" and "next" affordances.

Do you have suggestions for what additional clarity I could provide?